test/test_client.py

```diff
@@ -696,21 +707,22 @@ def test_no_creds_needed(self):
         @httmock.all_requests
         def auth_response(url, request):
             self.assertEqual(urllib.parse.urlunsplit(url),
-                             'https://auth.taskcluster.net/v1/clients/abc')
+                             'https://tc.my.com/api/auth/v1/clients/abc')
             self.failIf('Authorization' in request.headers)
             headers = {'content-type': 'application/json'}
             content = {"clientId": "abc"}
             return httmock.response(200, content, headers, None, 5, request)
 
         with httmock.HTTMock(auth_response):
-            client = subject.Auth({"credentials": {}})
+            client = subject.Auth({"rootUrl": "https://tc.my.com", "credentials": {}})
```

Comment:
Since this domain does resolve

```
[jhford@jhford ~]$ curl -I tc.my.com
HTTP/1.1 301 Moved Permanently
<snip>
Location: https://tc.my.com/
<snip>
```

could we make it a subdomain of example.com?

Reply:
Looks like it does resolve!  Sure, we should probably use the testing rootUrl anyway.